src/VirtualMonoRepo/source-mappings.json

```diff
@@ -150,7 +150,9 @@
         },
         {
             "name": "sourcelink",
-            "defaultRemote": "https://github.com/dotnet/sourcelink"
+            "defaultRemote": "https://github.com/dotnet/sourcelink",
+            // Migrated onto the flat flow
+            "disableSynchronization": true
```

Comment:
It is not obvious why this is being done as part of the rebootstrap?

Reply:
This had to be changed in https://github.com/dotnet/sdk/commit/6bca3a3355975b2cfa345d3e1194d9a07d78be71 and the re-bootstrap now allows to bring this back to the previous state.